detox/test/e2e/detox.config.js

```diff
@@ -27,6 +27,7 @@ const config = {
     init: {
       exposeGlobals: true
     },
+    optimizeAppInstall: Boolean(process.env.CI),
```

Comment:
should be true on Android and false on iOS..

Reply:
(`process.env.PLATFORM === 'android' && Boolean(process.env.CI) `)

Reply:
Why? If it were implemented on iOS, why wouldn't we want it to work?

Reply:
But it isn't implemented in iOS...